### PR TITLE
Adds more "curated" version of human randomization (for humonkeys, corpse spawners, and fresh characters)

### PR DESCRIPTION
**Pull request by MrMelbert**
## About The Pull Request

Basically, adds a version of `randomize_human` that's a tad more curated, IE, primarily picks results to create a more "coherent" result than full randomization. 

Shown here: Humans, Lizards, Felinids. Looks a bit boring, but that's the point.

image

Non-humans I left almost entirely untouched, as they generally form a more "coherent" mob from pure randomization already. 

## Why It's Good For The Game

Basically just aiming to perform a more "consistent" style for humonkeys and corpses. 
For monkeys, it doesn't make the most sense when they pop up with a giant red afro. 
For corpses, it's a bit hard to feel immersed in the ruins when they've all got pink and green mustaches.

## Changelog

:cl: Melbert
add: Humonkeys and random corpse spawns now look more... human.
/:cl:

### CODE REVIEW

**Comment by MTandi** [comment]
Thank you so much! It felt that I'm creating some freak show party by spraying a bunch of monkey cubes with extinguisher.

**Comment by Singul0** [comment]
great for ruin corpses

File: code/modules/admin/create_mob.dm
Comment (AdipemDragon) [inline code/modules/admin/create_mob.dm:1]: image
was kind bleh that the color selection wasn't put down in any visible sense so I tried my best to do so myself, left side is arranged in the order the list is shown here, the other is me slapping the colors in a loose ramp
 unsure how to feel about the several bunches of Very simmilar colors and also the seeming? lack? of graying hair colors?

Comment (MrMelbert) [reply code/modules/admin/create_mob.dm:1]: I literally just googled "Natural hair color hex palette" and picked the first 5, if you have a better range of colors feel free to drop it here

Comment (MrMelbert) [reply code/modules/admin/create_mob.dm:1]: Also it's visible in the code so that's mb
image

Comment (AdipemDragon) [reply code/modules/admin/create_mob.dm:1]: image

Maybe? Possibly?

Comment (AdipemDragon) [reply code/modules/admin/create_mob.dm:1]: Well I can't figure out how to do one of those reviews where people directly modify what's being pr'd and append it to this chain of comments specifically so lemme list the extra hex's that I added to the right in the image there:

"#41250C", "#3B3831",
"#863019", "#544C49",
"#A4381C", "#A29A96",
"#9F550E", "#C0BAB7",
"#B17B41", "#EFE5E4",
"#FFF2D6", "#F7F3F1",

like, the colors you picked are Fine, i just think there could be still more variance even with the "natural" constraint

**Comment by atlasle** [comment]
could there still be like a 5% chance that the old system is used?

File: code/modules/admin/create_mob.dm [hunk 1]
```diff
@@ -31,3 +31,41 @@
 	// Snowflake for Ethereals
 	human.updatehealth()
 	human.updateappearance(mutcolor_update = TRUE)
+
+/**
+ * Randomize human but attempts to produce a ""sane"" result
+ */
+/proc/randomize_human_sane(mob/living/carbon/human/human, randomize_mutations = FALSE)
+	var/static/list/nautural_hair_colors = list(
```
Comment (ArcaneMusic) [inline code/modules/admin/create_mob.dm:39]: natural vs nautural

**Comment by carlarctg** [comment]
honestly should be default char randomization too

**Comment by Mothblocks** [comment]
> could there still be like a 5% chance that the old system is used?

Would negate the entire point that they look hideous

**Comment by san7890** [comment]
undraft when CI passing

image

File: code/modules/admin/create_mob.dm [hunk 2]
```diff
@@ -31,3 +31,42 @@
...
+/proc/randomize_human_normie(mob/living/carbon/human/human, randomize_mutations = FALSE)
+	var/static/list/natural_hair_colors = list(
+		"#111111", "#362925", "#3B3831", "#41250C", "#412922",
+		"#544C49", "#583322", "#593029", "#703b30", "#714721",
+		"#744729", "#74482a", "#7b746e", "#855832", "#863019",
+		"#8c4734", "#9F550E", "#A29A96", "#A4381C", "#B17B41",
+		"#C0BAB7", "#EFE5E4", "#F7F3F1", "#FFF2D6", "#a15537",
+		"#a17e61", "#b38b67", "#ba673c", "#c89f73", "#d9b380",
+		"#dbc9b8", "#e1621d", "#e17d17", "#e1af93", "#f1cc8f",
+		"#fbe7a1",
```
Comment (JohnFulpWillard) [inline code/modules/admin/create_mob.dm:49]: Why not have/use defines?

Comment (MrMelbert) [reply code/modules/admin/create_mob.dm:49]: i don't really want to make "color_light_brunette" "color_lighter_brunette" "color_dark_brunette" "color_somewhat_brunette" "color_brunette"
